docroot/themes/humsci/su_humsci_theme/scss/components/atoms/_contextual_links.scss

```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
+
+// Overlapping contextual links are difficult. Different links show at the top for different users.
+// Space them all out so we can see them all.
+$sel: '';
+@for $i from 0 through 4 {
+  $sel: if($i == 0, '.contextual-region.paragraph', selector-nest($sel, '.contextual-region')) !global;
```

Comment:
Is the `!global` really necessary? It would be best to avoid if possible because you are already working on the global scope. 

Overall, this is a very concise function but I find it difficult to read. I don't often see this type of if/else statement. and generally, within Drupal, I usually see the long form as the standard practice. Eg: https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct with full line breaks and curly braces.

Reply:
No idea if `!global` is required. i took it from a blog post. 
that document is for php code standards. Not sure its applicable here

Reply:
I honestly don't really know what this does entirely. but it gave me the output i needed :laughing:

Reply:
The PHP doc was an example of a long-form statement. We use the long form in our PHP, JS, & SCSS. 

Here the function re-written to the long-form

```
@if ($i == 0) {
  $sel: '.contextual-region.paragraph';
} 
@else {
  $sel: $sel + ' .contextual-region'
}
```

Reply:
i thought we're not really enforcing long form anymore. it seems to be growing use in contrib and core. https://git.drupalcode.org/project/drupal/blob/8.8.x/core/lib/Drupal/Core/Database/Statement.php#L98
I think that documentation is outlining multiple scenario structures. not necessarily a binary condition.

Reply:
That ternary statement is different than the control structure here IMHO, but I won't block you on this project any further. We can pick up again on D8CORE or somewhere else.

Reply:
🤕